tests/ts/lsfd/mkfds-unix-in-netns

```diff
@@ -20,7 +20,8 @@ TS_DESC="UNIX sockets made in a differenct net namespace"
 . "$TS_TOPDIR"/functions.sh
 ts_init "$*"
 ts_skip_nonroot
-ts_skip_qemu_user
+
+. "$TS_SELF/lsfd-functions.bash"
```

Comment:
Not related to this patch set, but maybe we should use quotes for includes also in other tests to keep things consistent.

Reply:
BTW, for pidfs_open(2), there is "not available". (sorry for nitpicking:-)

Reply:
```
[jet@living]~/var/util-linux/tests/ts$ grep -r "^\." | grep '\$' | grep -v '"' | head -5
...
bitops/swapbytes:. $TS_TOPDIR/functions.sh
blkdiscard/offsets:. $TS_TOPDIR/functions.sh
blkid/dm-err:. $TS_TOPDIR/functions.sh
blkid/lowprobe-pt:. $TS_TOPDIR/functions.sh
blkid/md-raid0-whole:. $TS_TOPDIR/functions.sh
[jet@living]~/var/util-linux/tests/ts$ grep -r "^\." | grep '\$' | grep -v '"' | wc -l  
...
235
```

I will make another pull request for wrapping "$TS_..." with double quote characters.